### PR TITLE
Update rubocop

### PR DESCRIPTION
**Pull request by Averethel**

### CODE REVIEW

File: lib/lhs/data.rb [hunk 1]
```diff
@@ -18,7 +18,8 @@ class LHS::Data
   delegate :instance_methods, :items_key, :limit_key, :total_key, :pagination_key, to: :class
 
   # prevent clashing with attributes of underlying data
-  attr_accessor :_proxy, :_raw, :_parent, :_record, :_request, :_endpoint
+  attr_accessor :_proxy, :_parent, :_record, :_request, :_endpoint
+  attr_reader :_raw
```
Comment (spape) [inline lib/lhs/data.rb]: Line 25 uses the setter?!?!
https://github.com/local-ch/lhs/pull/297/files#diff-f65877927ba04cd8d2c1412a686c91c7R25

File: lib/lhs/pagination/base.rb [hunk 2]
```diff
@@ -27,7 +27,6 @@ def limit
     def offset
       data._raw.dig(*_record.pagination_key(:body)) || self.class::DEFAULT_OFFSET
     end
-    alias current_page offset
```
Comment (spape) [inline lib/lhs/pagination/base.rb]: wtf?!

File: lib/lhs/data.rb (hunk 1 above)
Comment (Averethel) [reply lib/lhs/data.rb]: Which is properly defined here: https://github.com/local-ch/lhs/blob/14dbc2d703e05317606334a202a091b2f4431008/lib/lhs/data.rb#L68

File: lib/lhs/pagination/base.rb (hunk 2 above)
Comment (Averethel) [reply lib/lhs/pagination/base.rb]: It's redefined here:
https://github.com/local-ch/lhs/blob/14dbc2d703e05317606334a202a091b2f4431008/lib/lhs/pagination/base.rb#L40

File: lib/lhs/data.rb (hunk 1 above)
Comment (spape) [reply lib/lhs/data.rb]: got it -.-

File: lib/lhs/pagination/base.rb (hunk 2 above)
Comment (spape) [reply lib/lhs/pagination/base.rb]: so, good catch then 🙈

**Pull request merged**